test/api/mailboxes-test.js

```diff
@@ -0,0 +1,186 @@
+/*eslint no-unused-expressions: 0, prefer-arrow-callback: 0, no-console:0 */
+
+/* globals before: false, after: false */
+
+'use strict';
+
+const supertest = require('supertest');
+const chai = require('chai');
+
+const expect = chai.expect;
+chai.config.includeStack = true;
+const config = require('wild-config');
+
+const server = supertest.agent(`http://127.0.0.1:${config.api.port}`);
+
+describe('Storage tests', function () {
```

Comment:
Is this a typo? Shouldn't it say Mailboxes tests? Applies to the user generation request below as well.